src/features/venuemap/components/VenueMapView/VenueMapView.tsx

```diff
@@ -82,6 +84,7 @@ export const VenueMapView: FunctionComponent<Props> = ({ padding }) => {
         mapPadding={padding}
         rotateEnabled={false}
         onRegionChangeComplete={handleRegionChangeComplete}
+        renderCluster={(props) => <VenueMapCluster {...props} />}
```

Comment:
```suggestion
        renderCluster={VenueMapCluster} />}
```
Est-ce que ça suffirait pas ?

Reply:
Nan ça ne fonctionne pas, `renderCluster` attend pas un Element mais plutôt `(cluster : any) => ReactNode` mais sans parler de typage cela casse côté app